### ISSUE TITLE
Import spark-xml in Jupyter Notebook

### ISSUE THREAD

**Issue by OXPHOS** [databricks/spark-xml#299]
Hello,

I am not sure whether my question is appropriate here. Interestingly, I didn't find any discussion about the question I have. I am not sure whether it is because no one else uses the package in my way or it's something really simple.

I had no trouble using `spark-xml` package by: 

`pyspark --packages com.databricks:spark-xml_2.10:0.4.1`

However, I'd like to directly import the package in Jupyter Notebook.

My Jupyter Notebook can start Spark session successfully and parse for example `.json` files with `SparkSession.read.json()`.  While so far I had no clue at all how to incorporate `spark-xml` in the kernel.

Basically, this is a question about how to import com.databricks package to Jupyter Notebook. Any help would be appreciated. Thanks so much!

**Comment by EmilioOldenziel** [comment]
Hi @OXPHOS,
This how I load the package into my notebooks using pyspark:
```
from os import environ
environ['PYSPARK_SUBMIT_ARGS'] = '--packages com.databricks:spark-xml_2.10:0.4.1 pyspark-shell' 
```
Hope this helps

**Comment by OXPHOS** [comment]
@EmilioOldenziel Works perfectly for me! Thanks so much!

**Comment by charu23** [comment]
Hi @EmilioOldenziel, still after loading the package as suggested by you above I am getting the same error as before, I am not sure if I am understanding this correct. Any help would be highly appreciated:


from os import environ
environ['PYSPARK_SUBMIT_ARGS'] = '--packages com.databricks:spark-xml_2.10:0.4.1 pyspark-shell'
from pyspark import SparkContext
from pyspark.sql import SQLContext
sqlContext = SQLContext(sc)
df = sqlContext.read.format('com.databricks.spark.xml').options(rootTag='Table').load(xml_file_location)
**ERROR:**
**java.lang.ClassNotFoundException: Failed to find data source: com.databricks.spark.xml. Please find packages at http://spark.apache.org/third-party-projects.html**

**Comment by bozopriester** [comment]
@charu...did you ever get that working?  I have the same issue and error.

**Comment by dheerajawale-svg** [comment]
> 
> 
> Hi @OXPHOS,
> This how I load the package into my notebooks using pyspark:
> 
> ```
> from os import environ
> environ['PYSPARK_SUBMIT_ARGS'] = '--packages com.databricks:spark-xml_2.10:0.4.1 pyspark-shell' 
> ```
> 
> Hope this helps

Is there any other way to do it?
